modules/analysis-common/src/test/resources/rest-api-spec/test/analysis-common/40_token_filters.yaml

```diff
@@ -10,6 +10,18 @@
     - length: { tokens: 1 }
     - match:  { tokens.0.token: Musee d'Orsay }
 
+    - do:
+        indices.analyze:
```

Comment:
I added this when I was debugging around for examples. It isn't strictly needed for this PR but I figure an extra example doesn't hurt.